### PR TITLE
astropy 4.0 moved iters -> maxiters

### PR DESCRIPTION
**Pull request by keflavich**
we should have caught this in a test...

### CODE REVIEW

File: astroquery/astrometry_net/core.py [hunk 1]
```diff
@@ -391,7 +391,7 @@ def solve_from_image(self, image_file_path, force_image_upload=False,
 
             print("Determining background stats", flush=True)
```
Comment (bsipocz) [inline astroquery/astrometry_net/core.py]: hmm, these prints should not be here.

**Comment by bsipocz** [comment]
I'm afraid it just shows that tests that need an API key and cannot be run from CI are equivalent to tests that don't exist. 
However, ideally, the non remote tests should have picked this up, too.

Would you care to add one that covers it?

**Comment by keflavich** [comment]
@bsipocz I propose we merge this as-is, then raise new issues to:

 * clean up (remove) print statements
 * bump the minimum required astropy version

**Comment by bsipocz** [comment]
@keflavich - The version bump needs to be done here, just to be sure master is consistent. I'm pushing commits to this branch.

**Comment by bsipocz** [comment]
Thanks @keflavich!

**Pull request merged**